app/src/main/kotlin/com/google/samples/apps/nowinandroid/MainActivity.kt

```diff
@@ -126,17 +133,25 @@ class MainActivity : ComponentActivity() {
                 onDispose {}
             }
 
-            CompositionLocalProvider(LocalAnalyticsHelper provides analyticsHelper) {
+            val appState = rememberNiaAppState(
+                windowSizeClass = calculateWindowSizeClass(this),
+                networkMonitor = networkMonitor,
+                userNewsResourceRepository = userNewsResourceRepository,
+                timeZoneMonitor = timeZoneMonitor,
+            )
+
+            val currentZoneId by appState.currentTimeZone.collectAsStateWithLifecycle()
```

Comment:
Nit: this variable name should likely be `currentTimeZone` with the API update

Reply:
Fixed